keps/prod-readiness/sig-node/1769.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 1769
+alpha:
+  - "@derekwaynecarr"
```

Comment:
You need PRR approve - can you please ask @deads2k or @johnbelamaric - I'm a bit oversubscribed...

Reply:
@deads2k @johnbelamaric Could I add one of you to carry out review and provide PRR approval?